server/api/themoviedb/interfaces.ts

```diff
@@ -1,3 +1,5 @@
+import { VideoType } from '../../models/Movie';
```

Comment:
It doesn't make sense to import a type from models into the typing for TMDB API. If anything it should be the other way around, or just have two mirrored, compatible types.

Reply:
agreed

Reply:
I've mirrored them for now , probably requires more clean up, as the Movie ones make sure its a youtube but the TV does not.

Reply:
I reverted this, no need for it.